lib/browser.js

```diff
@@ -34,7 +48,10 @@ function wrapFileWithStream (file) {
     throw new Error('You must provide a valid File or Blob object')
   }
 
-  return readStream(file).pipe(stream)
```

Comment:
In order to call the `abort()` method you need to return the FileStream instance. The `pipe()` method returns the destination only, that's why I had to change it not to be in a single line.
See: node_modules/filereader-stream/index.js#75